.pre-commit-hooks.yaml

```diff
@@ -4,3 +4,4 @@
     require_serial: true
     language: python
     types: [python]
+    additional_dependencies: [pipreqs, tomlkit, requirementslib, toml, pip-api]
```

Comment:
(pre-commit maintainer here) -- I would *not* suggest adding these to `additional_dependencies` by default -- this will slow down and bloat the installation when the consumer is not using these features -- the installation here should ~ideally be identical to just installing `isort`

plus there's potential for this to drift from the setuptools extras